app/client/components/delivery/records/deliveryRecordsProblemConfirmation.tsx

```diff
@@ -149,13 +163,29 @@ const DeliveryRecordsProblemConfirmationFC = (
             <InfoIconDark fillColor={palette.brand.bright} />
           </i>
           {deliveryProblemCredit?.showCredit
-            ? "Thank you for reporting your delivery problem. We will credit you for the affected issues and apologise for any inconvenience caused. We monitor these reports closely and use them to improve our service."
-            : "Your case is high priority. Our customer service team will try their best to contact you as soon as possible to resolve the issue."}
+            ? `Thank you for reporting your delivery problem${
+                deliveryAddressContext.address &&
+                deliveryAddressContext.productsAffected &&
+                deliveryAddressContext.productsAffected?.length > 0
```

Comment:
minor, but i doubt this `?` is necessary since you're doing the truthy check on the line before